migrations/sqls/20201028205336-followups-down.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE followUps;
```

Comment:
In situations where the up migrations fail, it's nice to make the down migrations flexible enough so that they don't also fail.

```suggestion
DROP TABLE IF EXISTS followUps;
```